aten/src/ATen/native/native_functions.yaml

```diff
@@ -9753,20 +9753,20 @@
 
 - func: inner.out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!)
 
-# torch.outer, alias for torch.ger
 - func: outer(Tensor self, Tensor vec2) -> Tensor
   variants: function, method
+  dispatch:
+    CompositeExplicitAutograd: outer
 
 - func: outer.out(Tensor self, Tensor vec2, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
+    CompositeExplicitAutograd: outer_out
```

Comment:
Should this have CompositeExplicitAutograd? Comparing this to "abs.out" as an analogy, "abs.out" does not have CompositeExplicitAutograd while "abs" does.

out= functions aren't differentiable in the first place, right?

Reply:
They are special cased in the codegen to not be. So I don't think this changes anything. I left it for consistency with the "ger" implementation. You think I should remove it?

Reply:
I think it is better without CompositeExplicitAutograd (because it is confusing for someone who isn't familiar to the code to read through: "why does this out= function have CompositeExplicitAutograd?") but I agree that this change is "no worse than before" so we can leave it as-is